hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -1105,6 +1114,28 @@ int regionIndexToEntityIndex(int region) {
     }
   }
 
+  static class ServerSsdLocalityCostFunction extends LocalityBasedCostFunction {
```

Comment:
What does this function do? Favor SSD? Needs comment.

Reply:
fix